contracts/XyParameterizer.sol

```diff
@@ -71,7 +71,7 @@ contract XyParameterizer {
         address _token,
         address _plcr,
         uint[] memory _parameters
-    ) internal {
+    ) public {
```

Comment:
this must be internal otherwise anyone can come and re-initialize the governance contract any time

Reply:
I see, I think I'll have to figure out that parameterizer test further since the test won't read the `init` function otherwise